tests/integration/smoke.bats

```diff
@@ -51,6 +34,20 @@ teardown() {
     [ "${lines[1]}" = "export FOO_BAR_BAM_SECRET" ]
 }
 
+@test "can seed a policy" {
+    run aomi seed
+    [ "$status" -eq 0 ]
+    run vault policies foo
+    [ "$status" -eq 0 ]
+}
+
+@test "can seed an app and user with builtin policy" {
+    run aomi seed
+    [ "$status" -eq 0 ]
+    run vault read -field=key auth/app-id/map/app-id/test
+    [ "$status" -eq 0 ]
+}
+
```

Comment:
Yeah there is an example in the smoke test for `aws_environment` https://github.com/Autodesk/aomi/blob/master/tests/integration/smoke.bats#L33